src/mantarray_desktop_app/main.py

```diff
@@ -138,6 +145,9 @@ def main(command_line_args: List[str], object_access_for_testing: Optional[Dict[
         # Tanner (5/20/22): not sure if this is actually logging anything since logging isn't configured yet
         logger.info(command_line_args)
 
+        # add filtering of /system_status logs
+        logger.addFilter(Filter_System_Status_Logs())
```

Comment:
A couple things here:
1. We already have a log formatter that we use. If we need to add any more filtering, it should be added to that one. I've mentioned this before, but it's very important that you read through and consider all of the existing code when adding new features to avoid adding something that is already present.
2. This solution does not filter out the log line that starts with `Response to HTTP Request in next log entry:`. You'll need to make sure we don't log that either for /system_status calls that return 200. Going of off the previous point, you will find that there is an existing function that can be modified to do this

Reply:
I implemented your comments but can't find the  `Response to HTTP Request in next log entry:`  `/system` log you mentioned. Is it in the filtered log file I posted?